composer/utils/object_store/uc_object_store.py

```diff
@@ -0,0 +1,96 @@
+from __future__ import annotations
+
+import logging
+import os
+import pathlib
+import uuid
+from typing import Callable
+
+from composer.utils.import_helpers import MissingConditionalImportError
+from composer.utils.object_store.object_store import ObjectStore
+
+log = logging.getLogger(__name__)
+
+
+class UCObjectStore(ObjectStore):
+    """Utility class for uploading and downloading data from Databricks Unity Catalog Volumes
+
+    Args:
+        uri (str): The Databricks UC Volume URI that is of the format
+            `dbfs:/Volumes/<catalog-name>/<schema-name>/<volume-name>/path`
+
+    """
+
+    def __init__(self, uri: str) -> None:
+        try:
+            import databricks
```

Comment:
Can we just replace `import databricks` with `from databricks.sdk import WorkspaceClient` since I don't see a usage of `databricks` directly ?